packages/react/src/components/CTA/TextCTA.js

```diff
@@ -75,23 +71,9 @@ TextCTA.propTypes = {
    * - [carbon-icons](https://www.npmjs.com/package/carbon-icons)!👀
    */
   type: PropTypes.oneOfType([
-    PropTypes.oneOf([
-      'jump',
-      'local',
-      'external',
-      'download',
-      'video',
-      'default',
-    ]),
+    PropTypes.oneOf(['jump', 'external', 'download', 'video']),
```

Comment:
This looks like a reduced set of icons, plus we aren't allowing for no icon. Is this part of the specs?